app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -204,7 +205,14 @@ class LuciBuildService {
         ),
       );
     }
-    await buildBucketClient.batch(BatchRequest(requests: requests));
+    const RetryOptions r = RetryOptions(maxAttempts: 3);
+    await r.retry(
+      () async {
+        await buildBucketClient.batch(BatchRequest(requests: requests));
+      },
+      retryIf: (Exception e) => e is BuildBucketException,
```

Comment:
This will only do retries with in a 2 second span. Should we modify `delayFactor` to be longer? Like 1s, 10s, 30s?

Reply:
Added a 2 sec delayFactor.